### PR TITLE
Exporter, makefile, readme, github actions

### PR DESCRIPTION
**Pull request by klatys**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+			PerPage: 100,
+		},
+	}
+	for {
+		repos, response, err := client.Repositories.ListByOrg(ctx, s.SourceIdent, &options)
+		if err != nil {
+			log.Errorf("could not fetch github response %s", err.Error())
+		}
+		for _, repo := range repos {
+			if repo.GetArchived() {
```
Comment (skooda) [inline main.go]: A little too much immersion, it's kinda hard to read. I'd move it out in a separate function.

File: main.go [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+		ListOptions: gitlab.ListOptions{
+			Page:    1,
+			PerPage: 100,
+		},
+	}
+
+	for {
+		groupProjects, response, _ := client.Groups.ListGroupProjects(s.SourceIdent, &options)
+		for _, project := range groupProjects {
+			if project.Archived {
```
Comment (skooda) [inline main.go]: Little bit deep again.

Separation to multiple well named functions can improve readability.

**Review by skooda — APPROVED** [review]
Just a minor improvements suggested.

File: main.go (hunk 1 above)
Comment (klatys) [reply main.go]: valid point, thanks! I've extracted repo prepare and get client methods

File: main.go (hunk 2 above)
Comment (klatys) [reply main.go]: valid point, thanks! I've extracted repo prepare and get client methods

**Review by mrlm-xyz — APPROVED** [review]

**Pull request merged**